privaterelay/templates/profile.html

```diff
@@ -73,7 +73,10 @@
       {% endif %}
       {% include "includes/banners.html" %}
 
-      {% include "includes/dashboard-filter.html" %}
+
+      {% if relay_addresses %}      
```

Comment:
Revised logic to hiding the filter entirely if the user does not have any aliases created. It currently shows as the user had no way to create their first alias with only the onboarding UI. 

Previous: 
![image](https://user-images.githubusercontent.com/2692333/127709484-23e0f972-8a47-4daa-9b1e-f0c4f7cc65c0.png)